Python/Module1_GettingStartedWithPython/Getting_Started_With_IDEs_and_Notebooks.md

```diff
@@ -49,7 +49,8 @@ There are many excellent IDEs that can be configured to work well with Python. T
 - works well out-of-the-box
 - long-supported by professionals and thus is very reliable
 - highly configurable
-- fully-featured
+- fully-featured, with an excellent debugger, context-dependent "intellisense", type-inference, and more
+- the free "community version" is extremely robust and feature-rich. 
```

Comment:
with this, maybe a con is that not everything is free (maybe with the acknowledgement that they may not be that heavily used when you're starting?)